pyaedt/hfss.py

```diff
@@ -38,7 +11,75 @@
 
 
 class Hfss(FieldAnalysis3D, object):
-    """HFSS object"""
+    """HFSS application interface.
```

Comment:
Again, I'd have changed it to """Hfss class.`` How would you know to put application interface?

Reply:
Hfss is a known solver or component of AEDT (at least I know that much).  We're telling the user that we've exposed an interface to it.